### DISCUSSION TITLE
Styling the colums header

### DISCUSSION THREAD

**Discussion by Yitavg** [KevinVandy/material-react-table#323] [Q&A]
Hi,
I have an issue styling the column header of the react table.
I used the following prop: 
```
  muiTableHeadCellProps={{
          sx: {
            background: backgroundColor,
            border: "2px solid red" 
          },
```
and got this result:
![tableCellHeader](https://user-images.githubusercontent.com/105783700/212534831-76ddc5fd-40a7-4730-ab90-398ad8c6a2c8.png)

As you can see, there is white background on each side between the header and the border.
What prop do I use to overwrite it? I literally tried every prop in the docs.
thanks

**Comment by KevinVandy** [comment]
The code is correct. You're pretty much following [this example](https://www.material-react-table.dev/?path=/story/styling-style-table-head-cells--style-all-mui-table-head-cell). I'm guessing you just passed an invalid color in.
